app/models/job/import_bank_file_event.rb

```diff
@@ -0,0 +1,17 @@
+module Job
+  class ImportBankFileEvent
+    @queue = :import_bank_file_events
+    def self.perform(import_bank_file_id, event_name)
+      import_bank_file = ImportBankFile.find(import_bank_file_id)
+      if ImportBankFile::VALID_EVENTS.include?(event_name)
```

Comment:
`VALID_EVENTS` is just here to make it little less insecure to call `#send(event_name)`